src/main/java/br/edu/restinga/ifrs/adotemeau/services/AnimalTemperamentService.java

```diff
@@ -0,0 +1,53 @@
+package br.edu.restinga.ifrs.adotemeau.services;
+
+import java.util.List;
+import javax.transaction.Transactional;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import br.edu.restinga.ifrs.adotemeau.http.dtos.AnimalTemperamentDTO;
+import br.edu.restinga.ifrs.adotemeau.models.AnimalTemperament;
+import br.edu.restinga.ifrs.adotemeau.repositories.AnimalTemperamentRepository;
+
+@Service
+public class AnimalTemperamentService {
+
+    @Autowired
+    private AnimalTemperamentRepository aTempRepository;
+    
+    //Cadastra um novo temperamento → create
```

Comment:
Comentário é bom somente quando é algo extremamente necessário. Este método está bem limpo e claro, como também, o seu nome já indica qual é a sua funcionalidade. Seria bom remover todos os comentários dessa classe...
```suggestion
    //Cadastra um novo temperamento → create
    //Lista todos os temperamentos → findAll
    //Atualiza um temperamento → update(int id)
    //Desativar ou ativar um temperamento. Nesse caso, a service deve receber true ou false → changeStatus(String action)
```